images/opensuse/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Default to tumbleweed distro
+ARG RELEASE=tumbleweed
+
+FROM opensuse/${RELEASE}
+
+# Install common utilities
+RUN zypper -n install \
+        iproute \
+        iputils \
+        openssh \
+        net-tools \
+        systemd-sysvinit \
+        udev \
+        sudo \
+        wget && \
+    zypper clean --all
+
+RUN systemctl enable sshd
```

Comment:
IIRC `systemctl` won't work during the image build (as it requires `systemd` to be running as PID 1). To work around this, create the symlink manually here (`systemctl enable sshd` on a host will show the source and target).

Reply:
I tested it locally and it's working for me

```
All repositories have been cleaned up.
Removing intermediate container 36876475bd36
 ---> 5ce4c71f7f9b
Step 4/5 : RUN systemctl enable sshd
 ---> Running in a26e0fbb9803
Removing intermediate container a26e0fbb9803
 ---> 896b98b208f7
```
anyway, I can change it if you want

Reply:
If it works for both `leap` and `tumbleweed` I'm fine with that, no need to change :+1:

Reply:
You could add a comment above that command for brevity, then I think we're ready to merge